container/root/etc/nginx/sites-available/default

```diff
@@ -1,6 +1,15 @@
 server {
   listen 8080;
 
+  #ssl on;
+  #ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
please remove TLSv1 (and possibly 1.1) from the list

Reply:
https://blog.pcisecuritystandards.org/are-you-ready-for-30-june-2018-sayin-goodbye-to-ssl-early-tls